### PR TITLE
Apply basic styling for the landing page

### PR DESCRIPTION
**Pull request by mpatterson88**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -2,21 +2,46 @@ import * as React from 'react';
...
+                        </div>
+                        <div id="atom">
+                            <img src={atom}/>
+                        </div>
+                    </Flex>
                 </div>
+
+                <div className="vert-spacer"/>
+
+                <Flex column={true} alignItems="center">
```
Comment (thinders) [inline src/App.tsx:28]: What's going on here: `column={true}`?

File: src/App.test.tsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
 import App from './App';
-import { shallow } from 'enzyme';
+import { render } from 'enzyme';
 
 it('renders without crashing', () => {
-  const app = shallow(
+  const app = render(
```
Comment (thinders) [inline src/App.test.tsx:6]: Won't this render the full app causing the `JamText` component to call out to our backend?

File: src/App.tsx (hunk 1 above)
Comment (mpatterson88) [reply src/App.tsx:28]: https://www.npmjs.com/package/react-flex

Comment (thinders) [reply src/App.tsx:28]: 👌

Comment (mpatterson88) [reply src/App.tsx:28]: It's an npm library that @tleyda had added. But we are considering ripping it out and just using normal css...

File: src/App.test.tsx (hunk 2 above)
Comment (mpatterson88) [reply src/App.test.tsx:6]: Potentially. I believe it only renders the dom but doesn't make the components fully functional, not sure. Either way, this version doesn't have any of the real components in it. So it works for now and once tas tries to merge he will have to deal with it :)

Comment (mpatterson88) [reply src/App.test.tsx:6]: Is there another way that will actually render all the components without making backend calls? With the new "Flex" components you don't get any content with the "shallow" command. But maybe that is another reason to move away from the react-flex library.

**Pull request merged**